project_event/i18n/fr.po

```diff
@@ -303,8 +308,9 @@ msgstr "Modèles d'événements"
 
 #. module: project_event
 #: model:ir.model.fields,field_description:project_event.field_project_event_wizard_name
+#: model:ir.ui.view,arch_db:project_event.project_event_tree
 msgid "Event Title"
-msgstr "Titre d'événement"
+msgstr "Titre de l'Évènement"
```

Comment:
@jananjoy Titre d'événement

Reply:
@rimbendhaou : Update  the  translation to the fr.po file